Source/Core/Core/PowerPC/Jit64IL/IR_X86.cpp

```diff
@@ -412,6 +423,8 @@ static X64Reg regBinLHSReg(RegInfo& RI, InstLoc I)
 	return reg;
 }
 
+// Clear any registers which end their lifetime at I
+// Don't use this for special instructions like memory load/stores
```

Comment:
Could you maybe explain what "special instruction" means here? I guess it's not immediately obvious to me what makes loads/stores special, or what non-load/store things might qualify as special and why.